src/components/ec-table-head/ec-table-head.vue

```diff
@@ -79,7 +82,23 @@ export default {
       this.$emit('sort', column.name);
     },
     getWidthStyle(column) {
-      return column && column.minWidth ? { minWidth: column.minWidth } : null;
+      let widthStyle = {};
```

Comment:
This is too much calculation in the JS for CSS. minWidth is okay here because it cannot be set via CSS, but rest of it can. Set a modifier class if there is a maxWidth and use ellipsis mixin in CSS.

Reply:
Updated. Now the ellipsis mixin is used instead.